File: packages/java/parser-jvm-plugin-backbone/src/test/java/com/vaadin/hilla/parser/plugins/backbone/multiendpoints/MultiEndpointsBazEndpoint.java
package com.vaadin.hilla.parser.plugins.backbone.multiendpoints;

import com.vaadin.hilla.Endpoint;

@Endpoint
public class MultiEndpointsBazEndpoint {
    public String getBaz() {
        return "baz";
    }

    public MultiEndpointsSharedModel getShared() {
        return new MultiEndpointsSharedModel();
    }
}

File: packages/java/typescript-generator/src/test/resources/com/vaadin/hilla/parser/plugins/backbone/multiendpoints/snapshots/MultiEndpointsBazEndpoint.ts
import type { EndpointRequestInit } from '@vaadin/hilla-frontend';
import type MultiEndpointsSharedModel from './com/vaadin/hilla/parser/plugins/backbone/multiendpoints/MultiEndpointsSharedModel.js';
import client from './connect-client.default.js';

export async function getBaz(init?: EndpointRequestInit): Promise<string | undefined> {
  return client.call('MultiEndpointsBazEndpoint', 'getBaz', {}, init);
}

export async function getShared(init?: EndpointRequestInit): Promise<MultiEndpointsSharedModel | undefined> {
  return client.call('MultiEndpointsBazEndpoint', 'getShared', {}, init);
}
